po/io.github.alainm23.planify.pot

```diff
@@ -334,7 +334,7 @@ msgstr ""
 
 #: src/Util.vala:1165
 msgid ""
-"To reorder your list, tap amd hold a to-do, then drag it to where it shpuld "
+"To reorder your list, tap and hold a to-do, then drag it to where it should "
```

Comment:
Edit source code instead of translations

Reply:
sorry! unfamiliar with the project and admittedly didn't look hard enough for the correct file. I've edited the source file now